--- a/akregator/src/subscription/subscriptionlistmodel.cpp
+++ b/akregator/src/subscription/subscriptionlistmodel.cpp
@@ -245,6 +245,9 @@
     }
 
     const Akregator::TreeNode *const parentNode = nodeForIndex(parent, m_feedList.data());
+    if (!parentNode) {
+        return QModelIndex();
+    }
     const Akregator::TreeNode *const childNode = parentNode->childAt(row);
     return  childNode ? createIndex(row, column, childNode->id()) : QModelIndex();
 }
