From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Wed, 13 Apr 2016 12:09:08 +0200
Subject: debianize_akregator_default

---
 akregator/src/akregator_part.cpp | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/akregator/src/akregator_part.cpp b/akregator/src/akregator_part.cpp
index 1841d2c..89509a6 100644
--- a/akregator/src/akregator_part.cpp
+++ b/akregator/src/akregator_part.cpp
@@ -183,6 +183,21 @@ static QDomDocument createDefaultFeedList()
     frenchKde.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("https://fr.planetkde.org/rss20.xml"));
     frenchFolder.appendChild(frenchKde);
 
+    // Debian feed(s)
+    QDomElement debianFolder = doc.createElement(QStringLiteral("outline"));
+    debianFolder.setAttribute(QStringLiteral("text"), QStringLiteral("Debian"));
+    body.appendChild(debianFolder);
+
+    QDomElement pland = doc.createElement(QStringLiteral("outline"));
+    pland.setAttribute(QStringLiteral("text"), i18n("Planet Debian"));
+    pland.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("http://planet.debian.org/rss20.xml"));
+    debianFolder.appendChild(pland);
+
+    QDomElement dtimes = doc.createElement(QStringLiteral("outline"));
+    dtimes.setAttribute(QStringLiteral("text"), i18n("Debian News"));
+    dtimes.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("http://www.debian.org/News/news"));
+    debianFolder.appendChild(dtimes);
+
     return doc;
 }
 }
