From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Wed, 13 Apr 2016 12:09:08 +0200
Subject: debianize_akregator_default

---
 akregator/src/akregator_part.cpp | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/akregator/src/akregator_part.cpp b/akregator/src/akregator_part.cpp
index bf6a913..0a66bb8 100644
--- a/akregator/src/akregator_part.cpp
+++ b/akregator/src/akregator_part.cpp
@@ -176,6 +176,20 @@ static QDomDocument createDefaultFeedList()
     spanishKde.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("http://planet.kde-espana.org/atom.xml"));
     spanishFolder.appendChild(spanishKde);
 
+    QDomElement debianFolder = doc.createElement(QStringLiteral("outline"));
+    debianFolder.setAttribute(QStringLiteral("text"), QStringLiteral("Debian"));
+    body.appendChild(debianFolder);
+
+    QDomElement pland = doc.createElement(QStringLiteral("outline"));
+    pland.setAttribute(QStringLiteral("text"), i18n("Planet Debian"));
+    pland.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("http://planet.debian.org/rss20.xml"));
+    debianFolder.appendChild(pland);
+
+    QDomElement dtimes = doc.createElement(QStringLiteral("outline"));
+    dtimes.setAttribute(QStringLiteral("text"), i18n("Debian News"));
+    dtimes.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("http://www.debian.org/News/news"));
+    debianFolder.appendChild(dtimes);
+
     return doc;
 }
 }
