Index: kdepim/akregator/src/akregator_part.cpp
===================================================================
--- kdepim.orig/akregator/src/akregator_part.cpp	2015-02-23 16:12:12.558279852 +0100
+++ kdepim/akregator/src/akregator_part.cpp	2015-02-23 16:18:15.791768443 +0100
@@ -184,6 +184,30 @@
         dtimes.setAttribute("xmlUrl","http://www.debian.org/News/news");
         debianFolder.appendChild(dtimes);
 
+        QDomElement kubuntuFolder = doc.createElement( "outline" );
+        kubuntuFolder.setAttribute("text","Kubuntu");
+        body.appendChild(kubuntuFolder);
+
+        QDomElement kuplanet = doc.createElement( "outline" );
+        kuplanet.setAttribute("text",i18n("Planet Ubuntu"));
+        kuplanet.setAttribute("xmlUrl","http://planet.ubuntu.com/rss20.xml");
+        kubuntuFolder.appendChild(kuplanet);
+
+        QDomElement kufridge = doc.createElement( "outline" );
+        kufridge.setAttribute("text",i18n("Ubuntu Fridge"));
+        kufridge.setAttribute("xmlUrl","http://feeds.feedburner.com/ubuntu-fridge");
+        kubuntuFolder.appendChild(kufridge);
+
+        QDomElement kutimes = doc.createElement( "outline" );
+        kutimes.setAttribute("text",i18n("Kubuntu.org News"));
+        kutimes.setAttribute("xmlUrl","http://www.kubuntu.org/news/feed");
+        kubuntuFolder.appendChild(kutimes);
+
+        QDomElement kuwire = doc.createElement( "outline" );
+        kuwire.setAttribute("text",i18n("Kubuntu Wire"));
+        kuwire.setAttribute("xmlUrl","http://wire.kubuntu.org/?feed=rss2");
+        kubuntuFolder.appendChild(kuwire);
+
         return doc;
     }
 }
