Index: kdepim/akregator/src/akregator_part.cpp
===================================================================
--- kdepim.orig/akregator/src/akregator_part.cpp	2015-02-23 14:27:47.364500343 +0100
+++ kdepim/akregator/src/akregator_part.cpp	2015-02-23 14:27:47.360500505 +0100
@@ -184,6 +184,30 @@
         dtimes.setAttribute("xmlUrl","http://www.debian.org/News/news");
         debianFolder.appendChild(dtimes);
 
+        QDomElement debianFolder = doc.createElement( "outline" );
+        debianFolder.setAttribute("text","Kubuntu");
+        body.appendChild(debianFolder);
+
+        QDomElement planet = doc.createElement( "outline" );
+        planet.setAttribute("text",i18n("Planet Ubuntu"));
+        planet.setAttribute("xmlUrl","http://planet.ubuntu.com/rss20.xml");
+        debianFolder.appendChild(planet);
+
+        QDomElement fridge = doc.createElement( "outline" );
+        fridge.setAttribute("text",i18n("Ubuntu Fridge"));
+        fridge.setAttribute("xmlUrl","http://feeds.feedburner.com/ubuntu-fridge");
+        debianFolder.appendChild(fridge);
+
+        QDomElement dtimes = doc.createElement( "outline" );
+        dtimes.setAttribute("text",i18n("Kubuntu.org News"));
+        dtimes.setAttribute("xmlUrl","http://www.kubuntu.org/news/feed");
+        debianFolder.appendChild(dtimes);
+
+        QDomElement wire = doc.createElement( "outline" );
+        wire.setAttribute("text",i18n("Kubuntu Wire"));
+        wire.setAttribute("xmlUrl","http://wire.kubuntu.org/?feed=rss2");
+        debianFolder.appendChild(wire);
+
         return doc;
     }
 }
