From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Wed, 13 Apr 2016 12:09:09 +0200
Subject: kubuntu_debianize_akregator_default

---
 akregator/src/akregator_part.cpp | 24 ++++++++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/akregator/src/akregator_part.cpp b/akregator/src/akregator_part.cpp
index 0a66bb8..7f79f6f 100644
--- a/akregator/src/akregator_part.cpp
+++ b/akregator/src/akregator_part.cpp
@@ -190,6 +190,30 @@ static QDomDocument createDefaultFeedList()
     dtimes.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("http://www.debian.org/News/news"));
     debianFolder.appendChild(dtimes);
 
+    QDomElement kubuntuFolder = doc.createElement(QStringLiteral("outline"));
+    kubuntuFolder.setAttribute(QStringLiteral("text"), QStringLiteral("Kubuntu"));
+    body.appendChild(kubuntuFolder);
+
+    QDomElement planet = doc.createElement(QStringLiteral("outline"));
+    planet.setAttribute(QStringLiteral("text"), i18n("Planet Ubuntu"));
+    planet.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("http://planet.ubuntu.com/rss20.xml"));
+    kubuntuFolder.appendChild(planet);
+
+    QDomElement fridge = doc.createElement(QStringLiteral("outline"));
+    fridge.setAttribute(QStringLiteral("text"), i18n("Ubuntu Fridge"));
+    fridge.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("http://feeds.feedburner.com/ubuntu-fridge"));
+    kubuntuFolder.appendChild(fridge);
+
+    QDomElement ktimes = doc.createElement(QStringLiteral("outline"));
+    ktimes.setAttribute(QStringLiteral("text"), i18n("Kubuntu.org News"));
+    ktimes.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("http://www.kubuntu.org/news/feed"));
+    kubuntuFolder.appendChild(ktimes);
+
+    QDomElement wire = doc.createElement(QStringLiteral("outline"));
+    wire.setAttribute(QStringLiteral("text"), i18n("Kubuntu Wire"));
+    wire.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("http://wire.kubuntu.org/?feed=rss2"));
+    kubuntuFolder.appendChild(wire);
+
     return doc;
 }
 }
