From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Wed, 13 Apr 2016 12:09:09 +0200
Subject: spambayes

===================================================================
---
 kmail/antispam-virus/kmail.antispamrc | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/kmail/antispam-virus/kmail.antispamrc b/kmail/antispam-virus/kmail.antispamrc
index 0671f3a..ad7b994 100644
--- a/kmail/antispam-virus/kmail.antispamrc
+++ b/kmail/antispam-virus/kmail.antispamrc
@@ -97,13 +97,13 @@ Ident=spambayes
 Version=1
 Priority=20
 VisibleName=SpamBayes
-Executable=sb_filter.py -h
+Executable=sb_filter -h
 URL=http://spambayes.sourceforge.net
 PipeFilterName=SpamBayes Check
-PipeCmdDetect=sb_filter.py
+PipeCmdDetect=sb_filter
 PipeCmdNoSpam=
-ExecCmdSpam=sb_filter.py -s
-ExecCmdHam=sb_filter.py -g
+ExecCmdSpam=sb_filter -s
+ExecCmdHam=sb_filter -g
 DetectionHeader=X-Spambayes-Classification
 DetectionPattern=spam
 DetectionPattern2=unsure
